hack/examples/java/reverser/function.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2017 The Nuclio Authors.
```

Comment:
Not sure about the whole storing jars in the source. Let alone calling then `nuclio-sdk-<version>` rather than `nuclio-sdk-java-<version>`. Let's discuss this